vue.config.js

```diff
@@ -20,7 +22,17 @@ module.exports = {
                     "nosort": false,
                     "prefixFileName": true,
                 }
-            })
+            }),
+            new CopyWebpackPlugin([
+                {
+                  from: './README.md',
```

Comment:
Change to include any `.md` files in project root, also formatting seems off.

Reply:
Formatting?? What formatting